travel-event-host-app/src/app/api/event/route.ts

```diff
@@ -0,0 +1,39 @@
+import { NextResponse } from 'next/server';
+import { connectMongoDB } from '@/lib/mongodb';
+import Event from '@/schemas/event';
+
+export async function POST(req: Request) {
```

Comment:
I think there is a way to type nextJS API requests https://stackoverflow.com/questions/69893369/how-to-add-typescript-types-to-request-body-in-next-js-api-route

Reply:
I think sanitize will be better